test/DebuggerCommon/step_in_from_interpreted_function_attach.js.dbg.baseline

```diff
@@ -2,19 +2,19 @@
   {
     "callStack": [
       {
-        "line": 27,
+        "line": 29,

```

Comment:
Another solution is enabling WScript.Platform only when a specific ch flag is used though. This way we won't have similar issues going forward